### DISCUSSION TITLE
Noob question, What code to run once you already loaded the model and started webGUI before?

### DISCUSSION THREAD

**Discussion by anywatt** [TheLastBen/fast-stable-diffusion#1656] [Q&A] [answered]
I've been using this notebook for a while but I might using wrong.
What I did to rerun the notebook is:
1. Mount GDrive
2. Run Install/Update AUTOMATIC1111 repo
3. Run Requirements
4. Run Model Download/Load (with Model_link input, Use_temp_storage: unchecked)
6. Start Stable-Diffusion

Is there a way to skip the "" if I already loaded them in the GDrive?
Or can I just hit the "Start Stable-Diffusion"?
Thank you.

**Comment by TheLastBen** [comment, marked as answer]
If you have the models in gdrive, specify that folder in the models cell (path to model) and it will load them, but you always have to run the cells from the first every time you fire up a colab instance

**Reply by anywatt** [reply]
Thank you.
It works for .safetensor Am I right?

**Reply by TheLastBen** [reply]
Yes

**Reply by brucez1234** [reply]
Follow up noob question:
I copied path to model to a folder in my gdrive containing multiple models I downloaded from CIVITAI; 3 of them are Loras I believe. All Safetensors files. However, when I try to start Stable Diffusion it gives me this:

Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 7, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'

How can we upload multiple models, specifically Loras? 

Thank you.


<img width="1139" alt="Screen Shot 2023-03-23 at 7 06 12 PM" src="https://user-images.githubusercontent.com/108761772/227383933-16b8e3c1-8869-42b5-a8b6-63fe5e965791.png">

**Reply by gilroff** [reply]
loras should go in lora folder.

**Comment by gilroff** [comment]
also, if u have loaded ur model once, u can skip running the model section next time if all ur models are in the same folder.
